### ISSUE TITLE
Question regarding the directories

### ISSUE THREAD

**Issue by kagestrife** [psychemedia/docker-containers#1]
In your blog, I read that you can actually put various projects (different folders with server.r/ui.r) in the same image. 
I cant find how to get to each folder via a dockerfile command. 
When I try your example in my local machine, I get the shiny example, which is in the /srv/shiny-server/ folder, instead of the example, located in /srv/shiny-server/shiny_demo. Is there a way to change the directory used when the image is created?
This may be a dumb question, but I just started using docker, so all help is more than welcome. Thanks in advance.

**Comment by psychemedia** [comment]
The statement `ADD shiny_demo /srv/shiny-server/shiny_demo` copies to the folder `shiny_demo` in the repo to the folder `/srv/shiny-server/shiny_demo` in the container image.

**Comment by kagestrife** [comment]
Yeah, but when I start the container it doesnt load your demo, it runs the shiny server example. I am running it locally, via the kitematic.

**Comment by psychemedia** [comment]
Ah  - ok - hmm - yes - the dockerfile just copies the folder, it doesn't actually change the command that's run by the rocker base container. A possibly incomplete example, in other words - though the [blogpost](https://blog.ouseful.info/2015/12/10/how-to-run-a-shiny-app-in-the-cloud-using-tutum-digital-ocean-and-docker-containers/) suggests it should work at `http://LOCAL_IP:PORT/shiny_demo/`

I get an error trying to start the application via the container, _though it works okay if I just try running the application from the code folder as normal via RStudio_.

Maybe `rocker` or `shiny-server` changed? <s>Or maybe the demo `shiny` code has rotted.</s>

I will add it to my list of rotted/broken blogposts... Not sure if/when I'll be able to get round to patching this though...

**Comment by psychemedia** [comment]
okay - I just had a quick look and I get the following log trace:

```
[2016-09-27 14:36:58.486] [INFO] shiny-server - Shiny Server v1.4.6.809 (Node.js v0.10.46)
[2016-09-27 14:36:58.488] [INFO] shiny-server - Using config file "/etc/shiny-server/shiny-server.conf"
[2016-09-27 14:36:58.513] [WARN] shiny-server - Running as root unnecessarily is a security risk! You could be running more securely as non-root.
[2016-09-27 14:36:58.516] [INFO] shiny-server - Starting listener on 0.0.0.0:3838
[2016-09-27 14:37:05.896] [INFO] shiny-server - Created bookmark state directory: /var/lib/shiny-server/bookmarks
[2016-09-27 14:37:05.902] [INFO] shiny-server - Created user bookmark state directory: /var/lib/shiny-server/bookmarks/shiny
[2016-09-27 14:37:05.978] [INFO] shiny-server - Error getting worker: Error: The application exited during initialization.
[2016-09-27 14:37:05.996] [INFO] shiny-server - Error getting worker: Error: The application exited during initialization.
```

No time to explore what that means though (I haven't used shiny for 6 months...(
